### ISSUE TITLE
terraform-apply fails obfuscating output values containing multi-line strings with "Error: Unable to process file command 'output' successfully."

### ISSUE THREAD

**Issue by timdhoffmann** [dflook/terraform-github-actions#219]
### Problem description

We have an output for an SSH private key defined as sensitive in our Terraform configuration (resolves to an SSH key, containing line feed `\n` escape sequences). Here, I inserted a multi-line dummy string literal containing the `\n` line feed escape sequence for testing purposes:
**`outputs.tf` configuration:**
```
# ...

output "tls_private_key" {
  # TODO: revert or fix after investigations.
  value     = "-----BEGIN TEST VALUE FOR MULTILINE STRING-----\nasldkfjasldkfjw2093j2039j230293jasdf\nasdfasdfasdfalskdjfaslkdfjasdlfkjasdlfkj23409234" 
  sensitive = true
}

# ...
```

### Expected

The output is obfuscated in GitHub action logs.

### Observed

The output is _not_, or only _partially_ obfuscated in GitHub action logs. Instead of obfuscating the full multi-line string, only the part up to the first escape sequence is obfuscated. The parts after each escape sequence are output to logs:
```
-----BEGIN TEST VALUE FOR MULTILINE STRING-----\nasldkfjasldkfjw2093j2039j230293jasdf\nasdfasdfasdfalskdjfaslkdfjasdlfkjasdlfkj23409234"
```
Is turned into
```
***
asldkfjasldkfjw2093j2039j230293jasdf
asdfasdfasdfalskdjfaslkdfjasdlfkjasdlfkj23409234
```

See the following details for the regular, as well as debug output.

*Note:* I also tested the behavior with a dummy string literal that doest _not_ contain the `\n` escape sequence, in which case the value is obfuscated in logs as expected!

**Regular output:**
```
// ...

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

Outputs:

//...
elasticsearch_password = <sensitive>
elasticsearch_username = "elastic"
//...
logshipper_integrations_vm_tls_private_key = <sensitive>
asldkfjasldkfjw2093j2039j230293jasdf
asdfasdfasdfalskdjfaslkdfjasdlfkjasdlfkj23409234
Error: Unable to process file command 'output' successfully.
Error: Invalid format 'asldkfjasldkfjw2093j2039j230293jasdf'
```

**Debug output:**
```
// ...

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

Outputs:

// ...
elasticsearch_password = <sensitive>
elasticsearch_username = "elastic"
// ...
logshipper_integrations_vm_tls_private_key = <sensitive>
##[debug]/tmp/github_pr_comment.stderr:::debug::['/usr/local/bin/github_pr_comment', 'status']
##[debug]/tmp/github_pr_comment.stderr:::debug::Backend fingerprint includes dict_keys(['storage_account_name', 'container_name', 'key', 'environment', 'endpoint', 'resource_group_name', 'msi_endpoint', 'subscription_id', 'tenant_id'])
##[debug]/tmp/github_pr_comment.stderr:::debug::Read comment from step_cache
##[debug]/tmp/github_pr_comment.stderr:::debug::Wrote comment to step_cache
##[debug] terraform output -json
::add-mask::***
::add-mask::***
asldkfjasldkfjw2093j2039j230293jasdf
asdfasdfasdfalskdjfaslkdfjasdlfkjasdlfkj23409234
##[debug]ls -la /github/workspace:total 44

// ...
```

Thanks for the great work, and thanks for looking into this!

### Terraform version

1.2.6

### Backend

Azure

**Comment by dflook** [comment]
Thanks for creating an iss @timdhoffmann, this should be fixed in v1.29.1

**Comment by timdhoffmann** [comment]
Thank you very much for the quick fix! Fix also confirmed from my testing.
